src/rust/dbv2/migrations/2024-02-08-135108_prices/up.sql

```diff
@@ -0,0 +1,26 @@
+-- Your SQL goes here
+CREATE TABLE aggregator.prices (
+    "market_id" NUMERIC(20,0),
+    "time" TIMESTAMPTZ,
+    "price" NUMERIC(20,0) NOT NULL,
+    "weight" NUMERIC(20,0) NOT NULL,
```

Comment:
This could be misleading due to the widespread use of TWAP (time-weighted average price)

I do think that weighting by size within the minute of calculation is the right approach, though, so how about renaming `sum_fill_size_1m_period`?